include/deal.II/grid/grid_tools.h

```diff
@@ -1239,6 +1239,34 @@ namespace GridTools
 
 
   /**
+   * Given an array of points use the global bounding box description obtained using
+   * GridTools::compute_mesh_predicate_bounding_boxto guess, for each of them,
+   * which process might own it.
+   *
+   * @param[in] global_bboxes : vector of bounding boxes describing the portion of
```

Comment:
@param[in] global_bboxes Vector

I believe that doxygen stylises all of the documentation, so this would lead to a floating ":" at the beginning of the parameter description. So it should be removed and the first letter capitalised. Same applies below.